client/src/containers/search-page/mapSearchData.ts

```diff
@@ -16,7 +16,7 @@ export const mapSearchData = (data: ISearchState['search']): ISearchPageProps['d
 			author: {
 				firstName: task?.user?.name || '',
 				lastName: task?.user?.surname || '',
-				link: '/',
+				link: ROUTES.Users + '/' + task?.user?.username || ROUTES.NotFound,
```

Comment:
Will this work?
don't you need to write a condition here like:
`link: task?.user?.username ? ROUTES.Users + '/' + task?.user?.username : ROUTES.NotFound`